### PR TITLE
[SYSE-87] Implements control points for pr, lts pr, tag

### PR DESCRIPTION
**Pull request by alephnull**
Also implements LTS support for api-tests.
Only PR scope is used at this point.

### CODE REVIEW

**Review by ermirizio — CHANGES_REQUESTED** [review]
Please add printing of the calculated versions after line 63

**Comment by alephnull** [comment]
> Please add printing of the calculated versions after line 63

See debug logs at https://github.com/TykTechnologies/tyk-pump/actions/runs/7335877466/job/19974434967?pr=757#step:7:41

**Comment by alephnull** [comment]
https://tyktech.atlassian.net/browse/SYSE-87?focusedCommentId=48101 blocks merging.

**Review by asutosh — DISMISSED** [review]

File: policy/templates/releng/.github/workflows/release.yml [hunk 1]
```diff
@@ -28,17 +28,45 @@ env:
...
+    steps:
+      - name: set params
+        id: params
+        env:
+          REPO: {{`${{ github.repository }}`}}
+          # Cover pull_request_target too
+          BASE_REF: {{`${{startsWith(github.event_name, 'pull_request') && github.base_ref || github.ref}}`}}
+          TAGS: {{`${{ needs.goreleaser.outputs.tags }}`}}
+          IS_PR: {{`${{startsWith(github.event_name, 'pull_request') && 'yes' }}`}}
+          IS_LTSPR: {{`${{(startsWith(github.event_name, 'pull_request') && endsWith(github.base_ref, '-lts')) && 'yes' }}`}}
```
Comment (ermirizio) [inline policy/templates/releng/.github/workflows/release.yml:52]:
```suggestion
          IS_LTS: {{`${{(startsWith(github.event_name, 'pull_request') && endsWith(github.base_ref, '-lts')) && 'yes' }}`}}
```

**Review by ermirizio — CHANGES_REQUESTED** [review]

**Review by ermirizio — APPROVED** [review]

**Pull request merged**